Modules/Loadable/Markups/VTKWidgets/vtkSlicerROIRepresentation2D.cxx

```diff
@@ -190,13 +189,21 @@ void vtkSlicerROIRepresentation2D::UpdateCubeSourceFromMRML(vtkMRMLMarkupsROINod
     this->SetROISource(cubeSource);
     }
 
+  // Add a small amount to each axis so that the ROI is still visible when the slice view is parallel to
+  // and intersecting with one of the faces.
+  double sliceNormal_XY[4] = { 0.0, 0.0, 1.0, 0.0 };
+  double sliceNormal_World[4] = { 0, 0, 1, 0 };
+  vtkMatrix4x4* xyToRAS = this->GetSliceNode()->GetXYToRAS();
```

Comment:
Usually we get the slice spacing from `this->SliceLogic->GetLowestVolumeSliceSpacing()[2]`. I'm wondering if we should use that here.

Reply:
Sure, done.